packages/create-astro/src/logger.ts

```diff
@@ -0,0 +1,141 @@
+import { bold, blue, dim, red, yellow } from 'kleur/colors';
+import { Writable } from 'stream';
+import { format as utilFormat } from 'util';
+
+type ConsoleStream = Writable & {
+	fd: 1 | 2;
+};
+
+function getLoggerLocale(): string {
+	const defaultLocale = 'en-US';
+	if (process.env.LANG) {
+		const extractedLocale = process.env.LANG.split('.')[0].replace(/_/g, '-');
+		// Check if language code is atleast two characters long (ie. en, es).
+		// NOTE: if "c" locale is encountered, the default locale will be returned.
+		if (extractedLocale.length < 2) return defaultLocale;
```

Comment:
```suggestion
		if (extractedLocale.length != 2) return defaultLocale;
```

Because could theoretically the code be `english` or the likes?

Reply:
I am not really sure, this code is just a straight copy of the logger implemented in astro core but with some parts removed that didn't seem necessary for `create-astro`, but I don't think that it would return anything like that, I am not very familiar with env variables though so I don't actually know.

Reply:
awesome